GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -1695,6 +1702,30 @@ private void InitializeComponent()
             this.ToolStripScripts.TabIndex = 2;
             this.ToolStripScripts.Text = "Scripts";
             // 
+            // gitextSettingsToolStripMenuItem
+            // 
+            this.gitextSettingsToolStripMenuItem.Image = global::GitUI.Properties.Images.GitExtensionsLogo16;
+            this.gitextSettingsToolStripMenuItem.Name = "gitextSettingsToolStripMenuItem";
+            this.gitextSettingsToolStripMenuItem.Size = new System.Drawing.Size(180, 22);
+            this.gitextSettingsToolStripMenuItem.Text = "Git Extensions &settings";
+            this.gitextSettingsToolStripMenuItem.Click += new System.EventHandler(this.GitextSettingsToolStripMenuItem_Click);
+            // 
+            // gitSettingsToolStripMenuItem
+            // 
+            this.gitSettingsToolStripMenuItem.Image = global::GitUI.Properties.Images.GitLogo16;
+            this.gitSettingsToolStripMenuItem.Name = "gitSettingsToolStripMenuItem";
+            this.gitSettingsToolStripMenuItem.Size = new System.Drawing.Size(180, 22);
+            this.gitSettingsToolStripMenuItem.Text = "&Git settings";
+            this.gitSettingsToolStripMenuItem.Click += new System.EventHandler(this.gitSettingsToolStripMenuItem_Click);
+            // 
+            // pluginsSettingsToolStripMenuItem
+            // 
+            this.pluginsSettingsToolStripMenuItem.Image = global::GitUI.Properties.Images.Plugin;
+            this.pluginsSettingsToolStripMenuItem.Name = "pluginsSettingsToolStripMenuItem";
+            this.pluginsSettingsToolStripMenuItem.Size = new System.Drawing.Size(180, 22);
+            this.pluginsSettingsToolStripMenuItem.Text = "&Plugins settings";
```

Comment:
```suggestion
            this.pluginsSettingsToolStripMenuItem.Text = "&Plugin settings";
```

The existing main menu item should be `Plugin &settings` (not capitalized, too).

![image](https://user-images.githubusercontent.com/36601201/210270331-89da9945-1732-4ca1-a7a0-a849a2468caa.png)

Reply:
Thinking about your suggestion, I think we should keep "Plugin*s* settings" and even update the "Plugins" menu entry to add an 's' because it's to manage the "plugins" and not a specific one. You even end up focus on the "Plugins" item in the treeview:
![image](https://user-images.githubusercontent.com/460196/212871831-692df4d1-0016-4b08-b676-e2da30443d4d.png)

Reply:
Done. It remains this integration test failing that I don't absolutely know what to do about 😢